source/libraries/ContractDeployer.ts

```diff
@@ -258,9 +278,24 @@ Deploying to: ${networkConfiguration.networkName}
         if (await universe.getTypeName_() !== stringTo32ByteHex("Universe")) {
             throw new Error("Unable to create genesis universe. Get type name failed");
         }
+
         return universe;
     }
 
+    private async migrateFromlegacyRep(): Promise<void> {
```

Comment:
Nit: shouldn't the 'l' in `migrateFromlegacyRep` be capitalized here?

Reply:
Will fix!